src/xdocs/config_filters.xml

```diff
@@ -648,6 +648,220 @@ public class UserService {
       </subsection>
     </section>
 
+    <section name="SuppressionXpathFilter">
+      <subsection name="Description">
+        <p>Since Checkstyle 8.4</p>
+        <p>
+          Filter <code>SuppressionXpathFilter</code> works as
+          <code>SuppressionFilter</code>.Additionally, filter
+          processes <code>suppress-xpath</code> elements,
+          which contains xpath-expressions. Xpath-expressions
+          are queries for suppressed nodes inside the AST tree.
+        </p>
+        <p>
+          Currently, filter supports the following
+          checks:
+        </p>
+        <ul>
+          <li>CyclomaticComplexityCheck</li>
+          <li>DeclarationOrderCheck</li>
+          <li>DefaultComesLastCheck</li>
+          <li>ExplicitInitializationCheck</li>
+          <li>FallThroughCheck</li>
+          <li>HiddenFieldCheck</li>
+          <li>IllegalCatchCheck</li>
+          <li>IllegalThrowsCheck</li>
+          <li>JavadocMethodCheck</li>
+          <li>JavadocVariableCheck</li>
+          <li>ImportControlCheck</li>
+          <li>LeftCurlyCheck</li>
+          <li>MethodParamPadCheck</li>
+          <li>MultipleVariableDeclarationCheck</li>
+          <li>NestedForDepthCheck</li>
+          <li>NestedIfDepthCheck</li>
+          <li>NestedTryDepthCheck</li>
+          <li>NPathComplexityCheck</li>
+          <li>OneStatementPerLineCheck</li>
+          <li>OuterTypeNumberCheck</li>
+          <li>RequireThisCheck</li>
+          <li>RightCurlyCheck</li>
+        </ul>
+      </subsection>
```

Comment:
please make a note what issues blocks support for other Checks, please add links to github issues.

Reply:
done